test/test.js

```diff
@@ -62,4 +62,28 @@ describe('Tests', () => {
 		expect(toMs('.62 sec')).to.equal(msInSec * 0.62, '.62 sec')
 		expect(toMs('0.54 sec')).to.equal(msInSec * 0.54, '0.54 sec')
 	})
+
+	it('should test and operator functionality', () => {
+		expect(toMs('1 day and 1 day')).to.equal((msInDay + msInDay), '1 day and 1 day')
```

Comment:
Can you test for situations where the value isn't just `1`? Also test for decimal value usages?

Reply:
You betcha, skipper